eng/native/gen-buildsys.cmd

```diff
@@ -80,6 +80,32 @@ if /i "%__Arch%" == "wasm" (
     set __ExtraCmakeParams=%__ExtraCmakeParams%  "-DCMAKE_SYSTEM_VERSION=10.0"
 )
 
+if /i "%__Os%" == "android" (
+    :: Keep in sync with $(AndroidApiLevelMin) in Directory.Build.props in the repository rooot
```

Comment:
please add eng/native/gen-buildsys.cmd to the list of paths to update in the root Directory.Build.props

Reply:
done!